Add New Themed Beatmap Packs news post (#2667)
ppy/osu-wiki: news/2019-10-20-beatmap-pack-management-update-1.md

---
layout: post
title: "Introducing Beatmap Pack Management"
date: 2019-10-20 19:00:00 +0000
---

An update of the Beatmap Pack Management project, including new pack releases!

![](/wiki/shared/news/banners/beatmap-packs.jpg)

> Our project is called Beatmap Pack Management and our prior focus is the development of Beatmap Packs. Many of you might not even know who we are and what we are doing, so I'd like to present to you some information about us:

Reviewer: ```suggestion
Our project is called Beatmap Pack Management and we are focused on the development of Beatmap Packs. Many of you might not even know who we are or what we are doing, so I'd like to present some information about us:
```

> I've created a small group of experienced users who are interested to work on handmade charts dedicated to a specific theme, idea, or share similarities. And so we've been actively working since July 2019 to provide new theme packs of decent quality with interesting concepts. The interest is very high and some noteworthy members of the community joined the group to help me substantiate their ideas in this project. We already have a few things to present to you, and I am sure you'll like them.

Reviewer: ```suggestion
I've created a small group of experienced users who are interested in working on handmade charts dedicated to specific themes, ideas, or beatmaps which share similarities. Since July 2019, we've been actively working to provide new theme packs of decent quality with interesting concepts. Interest in this project is very high and some noteworthy members of the community have joined the group to help me by contributing their ideas to this project. We already have a few things to present to you, and I am sure you'll like them.
```

### Alternator Style packs

*created by [lain](https://osu.ppy.sh/users/1650010)*  
*written by [Stefan](https://osu.ppy.sh/users/626907)*

> lain was passionated enough to work on the very first mapping-style-specific mappacks that has been released in the packlist, and focused on beatmaps where alternating is essential. He has worked on two versions, one with ranked maps and one purely with loved beatmaps. If you need maps to practice alternating or are good already and want to challenge yourself you should check these packs out!

Reviewer: ```suggestion
lain was passionate enough to work on the very first mapping style specific map packs released in the pack list. These first packs are focused on beatmaps where alternating is essential. He has worked on two versions, one with ranked beatmaps and one with loved beatmaps. If you need maps to practice alternating or are good already and want to challenge yourself you should check these packs out!
```

### osu!dan project

*written by [celerih](https://osu.ppy.sh/users/4696296)*

The osu!dan project is a brand new community created project that seeks to provide a sort of "campaign mode" to the game. Players will progress through levels of increasing difficulty in a skill of their choice, such as aim, stamina, accuracy, et cetera.

Each of the dans will contain 20 levels, each of increasing difficulty. In order to complete a level, the player must reach the requirements on the maps presented, like reaching a certain accuracy threshold or attaining a full combo. For now, the release of dan levels is purely for fun, a way to practice, or testing yourself, but a more official implementation is planned for the future. wink wink

> The first of the dans is now available to download on the beatmap packs page of this site. The skill tested in this first release is aim and is sure to push your circle clicking limits. The requirements for this dan are quite simple; just full combo 10 of the 11 maps in a level to progress to the next. Once again, this for now is just for your own practice, or to compare yourself with your friends—there is no implementation for keeping track of levels as of now.

Reviewer: ```suggestion
The first of the dans is now available to download on the beatmap packs page. The skill tested in this first release is aim and is sure to push your circle clicking limits. The requirements for this dan are quite simple; just full combo 10 of the 11 maps in a level to progress to the next. Once again, this for now is just for your own practice, or to compare yourself with your friends—there is no implementation for keeping track of levels as of now.
```

If you wish to join the osu!dan server to assist with playtesting future packs and stay up to date, [you can join the server!](https://discord.gg/b7cVmGd)

### October Loved packs

*written by [Stefan](https://osu.ppy.sh/users/626907)*

> As it is time again to bundle a few beatmaps, there are some new packs for loved beatmaps available since yesterday. I want to take the chance to explain why I do not bundle the latest maps from Loved already: Since there is always a small chance that a beatmap can become unloved (and therefore unranked) it's a bit tedious to keep track of these things and I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.

Reviewer: ```suggestion
As it is time again to bundle a few beatmaps, there are some new packs for loved beatmaps available since yesterday. I want to take the chance to explain why I did not bundle the latest maps from Loved already: Since there is always a small chance that a beatmap can become unloved (and therefore unranked) it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.
```

### What else to expect

> While I believe we're doing very well already, we're just about to start and currently I'm the only active person that coordinates the uploads, as well as the accessibility and maintenance of the packs so it's a bit slow. But I'm glad I could help a lot of people so far and that I have received so much great feedback. If you are interested to take a look in what we are working on, or if you have to report an error/bug in a pack you can join our project server and spectate our doings. Alternatively, if you are interested to work in the group you can join it too! Please check out the [thread](https://osu.ppy.sh/community/forums/topics/933379) for information on how to get to the server.

Reviewer: ```suggestion
While I believe we're doing very well already, we're just about to start and currently I'm the only active person that coordinates the uploads, as well as the accessibility and maintenance of the packs so it's a bit slow. But I'm glad I could help a lot of people so far and that I have received so much great feedback. If you are interested in taking a look at what we are working on, or if you have to report an error/bug in a pack, you can join our project server and spectate our doings. Alternatively, if you are interested to work in the group you can join it too! Please check out the [thread](https://osu.ppy.sh/community/forums/topics/933379) for information on how to get to the server.
```

> While I believe we're doing very well already, we're just about to start and currently I'm the only active person that coordinates the uploads, as well as the accessibility and maintenance of the packs so it's a bit slow. But I'm glad I could help a lot of people so far and that I have received so much great feedback. If you are interested to take a look in what we are working on, or if you have to report an error/bug in a pack you can join our project server and spectate our doings. Alternatively, if you are interested to work in the group you can join it too! Please check out the [thread](https://osu.ppy.sh/community/forums/topics/933379) for information on how to get to the server.

Reviewer 2: I I I I I I I

please stop referring to self. if it's not a team effort it's not a news post, sorry.

Reviewer 3: Stefan is the listed author of this post so this perspective should be fine. he says early on that the whole project being talked about is a team effort

Author: I'd have agree to peppy's comment if the Beatmap Pack Mangagement was standing as author at the end of the news post, so it would make more sense to not refer to myself. If that is still problematic I'm fine rewording the text to not refer to me.

> I can also recommend to follow [@osuMapPacks](https://twitter.com/osuMapPacks) on Twitter to stay updated with news, fixes, new packs and more. With that said, I can't really tell when the next news post will follow since we're a group that takes time to work on packs individually but you can still contact me for questions if you have any. And I hope you have an amazing day and download lotsa packs, thank you for reading.

Reviewer: ```suggestion
I also recommend following [@osuMapPacks](https://twitter.com/osuMapPacks) on Twitter to stay updated with news, fixes, new packs and more. With that said, I can't really say when the next news post will be since we're a group that takes time to work on packs individually, but you can still contact me for questions if you have any. I hope you have an amazing day and download lots of packs, thank you for reading.
```

> I can also recommend to follow [@osuMapPacks](https://twitter.com/osuMapPacks) on Twitter to stay updated with news, fixes, new packs and more. With that said, I can't really tell when the next news post will follow since we're a group that takes time to work on packs individually but you can still contact me for questions if you have any. And I hope you have an amazing day and download lotsa packs, thank you for reading.

Reviewer 2: twitter handle has not been approved for use by anyone here on the team

Author: So in another words; directing to the Twitter profile is not available here?

Author: Just to clarify: The twitter account is used by me and serves purely for tracking of new additions and major changes. Every smaller change will be logged and the log will be available at the end of month. The account is used for nothing else and no one else can access it, if that helps.

---
layout: post
title: "Introducing Beatmap Pack Management"
date: 2019-10-20 19:00:00 +0000
---

An update to the Beatmap Pack Management project, including new pack releases!

![](/wiki/shared/news/banners/beatmap-packs.jpg)

> Our project is called Beatmap Pack Management and we are focused on development of the Beatmap Packs. Many of you might not even know who we are or what we are doing, so I'd like to present some information about us:

Reviewer 4: ```suggestion
Our project is called the Beatmap Pack Management and we are focused on the development of Beatmap Packs. Many of you might not even know who we are or what we are doing, so I'd like to present some information about us:
```

Reviewer 5: don't think if improvement

Author: looks good to me..?

> I've created a small group of experienced users who are interested in working on handmade charts dedicated to specific themes, ideas, or beatmaps which share similarities. Since July 2019, we've been actively working to provide new theme packs of decent quality with interesting concepts. Interest in this project is very high and some noteworthy members of the community have joined the group to help me by contributing their ideas to this project. We already have a few things to present to you, and I am sure you'll like them.

Reviewer: `of decent quality` I suggest removing this. 
It makes it sound like "eh this is ok" rather than hyping it up as a cool new project. Focus on what's good about it!

Author: simplified that part

### Alternator Style packs

*created by [lain](https://osu.ppy.sh/users/1650010), [Uberzolik](https://osu.ppy.sh/users/1314547)*  
*written by [Stefan](https://osu.ppy.sh/users/626907)*  
Download [Ranked](https://osu.ppy.sh/beatmaps/packs/1699) | [Loved](https://osu.ppy.sh/beatmaps/packs/1700)  

> lain and Uberzolik were passionate enough to work on the very first mapping style specific beatmap packs ever released. These two packs are focused on beatmaps where alternating is essential. lain has worked on the ranked version while Uberzolik took care of the Loved version. If you need maps to practice alternating or are good at it already and want to challenge yourself, you should check these packs out!

Reviewer 4: ```suggestion
lain and Uberzolik were passionate enough to work on the very first mapping style specific beatmap packs ever released. These two packs are focused on beatmaps where alternating is essential. lain has worked on the ranked version while Uberzolik took care of the Loved version. If you need maps to practice alternating or you are good at it already and want to challenge yourself, you should check these packs out!
```

### Best of 2010–2011 pack

*created by [Dada](https://osu.ppy.sh/users/9119507), [\_p0ke\_](https://osu.ppy.sh/users/5434711), [Halfslashed](https://osu.ppy.sh/users/4598899), [Uberzolik](https://osu.ppy.sh/users/1314547), and [Cynplytholowazy](https://osu.ppy.sh/users/3901754)*  
*written by [\_p0ke\_](https://osu.ppy.sh/users/5434711)*  
[Download](https://osu.ppy.sh/beatmaps/packs/1733)  

Ever wanted to see what popular maps looked like in earlier eras of the game? Well now you can! Forged together by members of the Beatmap Pack Management team, you can check out some of the best maps that the game had to offer during 2010 and 2011 in a single pack. While there is a lot of great maps, the team didn't have enough room to include everything, so hopefully you can find something on this list that you will enjoy. Play through them all and experience a blast from the past!

### osu!dan project

*written by [celerih](https://osu.ppy.sh/users/4696296)*  

> The osu!dan project is a brand new community created project that seeks to provide a sort of "campaign mode" to the game. Players will progress through levels of increasing difficulty in a skill of their choice, such as aim, stamina, accuracy, et cetera.

Reviewer 4: ```suggestion
The osu!dan project is a brand new community created project that seeks to provide a "campaign mode" to the game. Players will progress through levels of increasing difficulty in a skill of their choice, such as aim, stamina, accuracy, et cetera.
```

Each of the dans will contain 20 levels, each of increasing difficulty. In order to complete a level, the player must reach the requirements on the maps presented, like reaching a certain accuracy threshold or attaining a full combo. For now, the release of dan levels is purely for fun, a way to practice, or testing yourself, but a more official implementation is planned for the future. wink wink

> The first of the dans is now available to download on the beatmap packs page. The skill tested in this first release is aim and is sure to push your circle clicking limits. The requirements for this dan are quite simple; just full combo 10 of the 11 maps in a level to progress to the next. Once again, this for now is just for your own practice, or to compare yourself with your friends—there is no implementation for keeping track of levels as of now.

Reviewer 6: hey, it says "is NOW available", not "is NOT available"

> The first of the dans is now available to download on the beatmap packs page. The skill tested in this first release is aim and is sure to push your circle clicking limits. The requirements for this dan are quite simple; just full combo 10 of the 11 maps in a level to progress to the next. Once again, this for now is just for your own practice, or to compare yourself with your friends—there is no implementation for keeping track of levels as of now.

Reviewer: and it's not available. that's the problem. the news post is currently misleading to users.

> The first of the dans is now available to download on the beatmap packs page. The skill tested in this first release is aim and is sure to push your circle clicking limits. The requirements for this dan are quite simple; just full combo 10 of the 11 maps in a level to progress to the next. Once again, this for now is just for your own practice, or to compare yourself with your friends—there is no implementation for keeping track of levels as of now.

Author: I've been waiting for the news post to be "ready to release" before I make the packs public. They're all ready to be released but I wanna hold them back until we're good with the news post.

> If you wish to join the osu!dan server to assist with playtesting future packs and stay up to date, [you can join the server!](https://discord.gg/b7cVmGd)

Reviewer 5: ```suggestion
If you wish to help with playtesting future packs and stay up to date, you can join [the official osu!dan Discord server](https://discord.gg/b7cVmGd) now!
```

> If you wish to help with playtesting future packs and stay up to date, you can join [the official osu!dan Discord server](https://discord.gg/b7cVmGd) now!

Reviewer 4: ```suggestion
If you wish to help with playtesting future packs and want to stay up to date, you can join the official [osu!dan Discord server](https://discord.gg/b7cVmGd) now!
```

### October Loved packs

*written by [Stefan](https://osu.ppy.sh/users/626907)*

> As it is time again to bundle a few beatmaps, there are some new packs for loved beatmaps available since yesterday. I want to take the chance to explain why I did not bundle the latest maps from Loved already: Since there is always a small chance that a beatmap can become unloved (and therefore unranked) it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.

Reviewer: ```suggestion
As it is time again to bundle a few beatmaps, there are some new packs for loved beatmaps available since mid October. I want to take the chance to explain why I did not bundle the latest maps from Loved already: Since there is always a small chance that a beatmap can become unloved it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.
```

`since yesterday.` isn't right when it's been over two weeks. 
`(and therefore unranked)` but they weren't ranked to begin with?

Author: Ah, that really sounds odd.

> As it is time again to bundle a few beatmaps, there are some new packs for loved beatmaps available since yesterday. I want to take the chance to explain why I did not bundle the latest maps from Loved already: Since there is always a small chance that a beatmap can become unloved (and therefore unranked) it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.

Reviewer 5: ```suggestion
As it is time to bundle a few beatmaps again, there are some new packs for Loved beatmaps available since yesterday. I want to take the chance to explain why I did not bundle the latest maps from Loved already - since there is always a small chance that a beatmap can get unloved (and therefore unranked) it's a bit tedious to keep track of these things. I decided to give maps at least a minimum of 10 months to stay in Loved, and to bundle them afterwards into packs. While it's no guarantee they will stay in Loved forever, it works for me and I'll go by that for now. Changes can always happen though and I am open for alternatives.
```

### What else to expect

> While I believe we're doing very well already, we're just about to start and currently I'm the only active person that coordinates the uploads, as well as the accessibility and maintenance of the packs so it's a bit slow. But I'm glad I could help a lot of people so far and that I have received so much great feedback. If you are interested in taking a look at what we are working on, or if you have to report an error/bug in a pack, you can join our project server and spectate our doings. Alternatively, if you are interested to work in the group you can join it too! Please check out the [thread](https://osu.ppy.sh/community/forums/topics/933379) for information on how to get to the server.

Reviewer 4: ```suggestion
While I believe we're doing very well already, we're just about to start. Currently, I'm the only active person that coordinates the uploads, as well as the accessibility and maintenance of the packs so it's a little slow. But I'm glad I could help a lot of people so far and that I have received so much great feedback. If you are interested in taking a look at what we are working on, or if you have to report an error/bug in a pack, you can join our project server and spectate our doings. Alternatively, if you are interested to work in the group you can join it too! Please check out the [forum thread](https://osu.ppy.sh/community/forums/topics/933379) for information on how to get to the server.
```

> I also recommend following [@osuMapPacks](https://twitter.com/osuMapPacks) on Twitter to stay updated with news, fixes, new packs and more. With that said, I can't really say when the next news post will be since we're a group that takes time to work on packs individually, but you can still contact me for questions if you have any. I hope you have an amazing day and download lots of packs, thank you for reading.

Reviewer 5: ```suggestion
I also recommend following [@osuMapPacks](https://twitter.com/osuMapPacks) on Twitter to stay up-to-date with news, fixes, new packs and more. With that said, I can't really say when the next news post will be, since we're a group that takes time to work on packs individually, but you can still contact me if you have any questions. I hope you have an amazing day and download lots of packs, thank you for reading!
```
